hackclub.com.yaml

```
@@ -1153,8 +1154,8 @@ calendar:
   value: ghs.googlehosted.com.
 camp: # christian on slack U0938MZG8Q6 poc is euan
   - ttl: 300
-    type: CNAME
-    value: c7fd0190e9c83af1.vercel-dns-017.com.
+    type: TXT
+    value: http://c7fd0190e9c83af1.vercel-dns-017.com.
```

no protocol on cnames
```suggestion
    value: c7fd0190e9c83af1.vercel-dns-017.com.
```